pimd/pim_cmd_common.c

```diff
@@ -3805,7 +3805,8 @@ void show_mroute(struct pim_instance *pim, struct vty *vty, pim_sgaddr *sg,
 
 static void show_mroute_count_per_channel_oil(struct channel_oil *c_oil,
 					      json_object *json,
-					      struct vty *vty)
+					      struct vty *vty,
+					      struct ttable *tt, char *table)
```

Comment:
You don't need to pass the table pointer here, it's not required.

Reply:
Fixed